<launch>

  <!--arg name="gazebo_world"  default="fmt_willowgarage" /-->
  <arg name="gazebo_world"  default="fmt_warehouse_karto" />

  <!-- true | false -->
  <arg name="gazebo_gui"    default="false" />

  <!-- turtlebot | rb1-->
  <arg name="robot_type"    default="turtlebot" />

  <!-- visualisation is not necessarily necessary -->
  <arg name="do_viz"        default="false" />




  <!-- launch world, robot, etc -->
  <include file="$(find relief_devel)/launch/avanti_simulation.launch">
    <arg name="gazebo_world"  value="$(arg gazebo_world)" />
    <arg name="gazebo_gui"    value="$(arg gazebo_gui)" />
    <arg name="robot_type"    value="$(arg robot_type)" />
    <arg name="do_viz"        value="$(arg do_viz)" />
    <arg name="num_cameras"   value="1" />
    <arg name="loc_alg"       value="mcl_ros" />
  </include>



  <!-- Launch logger-->
  <!-- Throw the robot at a random pose first, then move slightly, then
       capture the pose output by mcl_ros -->
  <include file="$(find cbgl_ros)/launch/mcl_logger_sim.launch">
    <arg name="gazebo_world" value="$(arg gazebo_world)" />
  </include>
</launch>
